addons/l10n_it_stock_ddt/i18n/it.po

```diff
@@ -261,8 +261,8 @@ msgstr "Data transporto"
 
 #. module: l10n_it_stock_ddt
 #: model:ir.model.fields.selection,name:l10n_it_stock_ddt.selection__stock_picking__l10n_it_transport_reason__substitution
-msgid "Substitution"
-msgstr "Sostituzione"
```

Comment:
It's not removed, it's modified.
I tried adding both and it fails. :(
Some hint on how to proceed?

![image](https://github.com/odoo/odoo/assets/1665365/40ef62b7-816a-4896-83b8-07598f8b71f4)
![image](https://github.com/odoo/odoo/assets/1665365/f9aa9723-3954-485f-b0d0-884e077a030d)
![image](https://github.com/odoo/odoo/assets/1665365/25301c69-6d01-4c8a-bcd7-2d9dff52fe61)

Reply:
If it is only used in a selection field, it is alright.
The issue is when the translation is used in views/xml because those will lose the translation if the module is not updated and keeps the old string; but the selection field doesn't need an update and only a restart of the server.